### DISCUSSION TITLE
Support

### DISCUSSION THREAD

**Discussion by 5axes** [5axes/UniversalCuraSettings#22] [Profile or Intent]
Easy support remove

minimum_support_area = 8
support_bottom_density = 8
support_bottom_distance = _layer_height
support_bottom_enable = False
support_bottom_enable = True
support_bottom_height = round((_layer_height*4),1)
support_brim_enable = True
support_brim_width = 3
support_enable = True
support_infill_rate = 7
support_interface_density = 30
support_interface_enable = True
support_interface_line_width = round((_line_width*1.3),1)
support_interface_pattern = 'zigzag'
support_interface_skip_height = _layer_height
support_join_distance = 3
support_material_flow = 70
support_offset = round((_layer_height*3),1)
support_pattern = 'lines'  # or zigzag
support_roof_density = 90
support_roof_enable = True
support_roof_height = round((_layer_height*6),1)
support_roof_offset = round((_layer_height*3),1)
support_structure = 'normal'
support_top_distance = _layer_height
support_type = 'buildplate'
support_wall_count = 0
support_xy_distance = round((_line_width*2),1)
support_xy_overrides_z = 'z_overrides_xy' # or xy_overrides_z
support_z_distance = round((_layer_height),1))

**Comment by luizbgomide** [comment]
I've had some issues with those settings when applied over Standard Quality Profile + Standard Universal Settinngs.

At first it was missing the roof interface setting, and even after it was enabled it was still generating supports in mid air:
![image](https://user-images.githubusercontent.com/18707781/153971881-b3a8f555-495e-4220-8407-62121e561a0a.png)

**Comment by 5axes** [comment]
Yes you are right the Gradual Infill Step is activated by default and with the "Support" Configuration you can have this result. I will chnage that for a next release. Thanks your your report.

Temporary solution : set the **Gradual Infill Step = 0**

**Comment by djarulakis** [comment]
Hi! y have a problem with the plugin, in Cura 5.2.1 the menu show nothing. 
![image](https://user-images.githubusercontent.com/39717148/208265173-b832eb6b-f600-4081-9f75-5c4e5af1eeca.png)
If I use the dropdown, it shows the options but y can see what is selected, or if is selected correctly.
is this a normal issue with the plugin 0.1.9? and in the description in the cura marketplace says that are 11 different presets, but y see only 5 in the dropdown menu.
thanks!!!

**Reply by 5axes** [reply]
https://github.com/5axes/UniversalCuraSettings/issues/36
